pkg/scalers/azure_eventhub_scaler_test.go

```diff
@@ -111,11 +112,11 @@ func TestGetUnprocessedEventCountInPartition(t *testing.T) {
 	t.Logf("EventHub has 1 message in partition 0 and 0 messages in partition 1")
 
 	eventHubKey := os.Getenv("AZURE_EVENTHUB_KEY")
-	storageConnectionString := os.Getenv("AZURE_STORAGE_CONNECTION_STRING")
+	storageConnectionString := os.Getenv("TEST_STORAGE_CONNECTION_STRING")
```

Comment:
Is there any reason for this change?

Reply:
Nope, it must have come from some sort of copy/paste activity because I absolutely don't recall having changed that one. I'll revert it to the original env variable...weird.